bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -136,7 +136,7 @@ <h4 class="heading-windows">{{_('Download Firefox <span>for Windows</span>') }}<
         <h4 class="heading-osx">{{_('Download Firefox <span>for macOS</span>') }}</h4>
         <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
 
-        {{ download_firefox(dom_id='download-other-platforms-modal', alt_copy=_('Free Download')) }}
+        {{ download_firefox(dom_id='download-other-platforms-modal', alt_copy=_('Free Download'), locale_in_transition=True) }}
```

Comment:
Not related to this change, but I noticed the dl buttons in the modal weren't including the locale so fixing that here.